src/coord/src/catalog/builtin.rs

```diff
@@ -1325,6 +1334,43 @@ lazy_static! {
 
         assert!(func_global_id_counter < 3000, "exhausted func global IDs");
 
+        // Ensure the IDs we assign are all unique:
```

Comment:
nit: if you could move this set of changes into its own commit I'd love that just because its logically distinct from everything else iuc. but also no pressure!

Reply:
Yep, I'll rebase/squash this test and the extra table into two separate commits.

Reply:
done - it's three separate commits now (the test, the table, and the web UI), so I'll merge it as a non-squash.